include/solana.hpp

```diff
@@ -301,6 +301,12 @@ struct GetSlotConfig {
  */
 void to_json(json &j, const GetSlotConfig &config);
 
+struct largestAccountsConfig {
```

Comment:
Should be `LargestAccountsConfig` right?

Reply:
Changed the struct name to `LargestAccountsConfig`.. Thanks